react-native-community/upgrade-helper
# Add initial support for `react-native-windows`

# Summary

This PR adds initial support for `react-native-windows` by accepting a `package` query string with `react-native` and `react-native-windows` as possible values, if none provided then `react-native` is used as default.

As for the diffs, for `react-native-windows` it uses https://github.com/acoates-ms/rnw-diff and for `react-native` it remains using https://github.com/react-native-community/rn-diff-purge.



## Test Plan

1. Access https://deploy-preview-273--upgrade-helper.netlify.app/?from=0.63.4&to=0.64.1;
2. Make sure that it loads `react-native` diff;
3. Access https://deploy-preview-273--upgrade-helper.netlify.app/?from=0.63.4&to=0.64.1&package=react-native;
2. Make sure that it loads `react-native` diff;
3. Access https://deploy-preview-273--upgrade-helper.netlify.app/?package=react-native-windows;
2. Make sure that it loads `react-native-windows` diff.

## Issues that need to be fixed 

- [x] For some reason there's a warning about `key` being duplicated with the value of `undefinedundefined` in `DiffSection`, only happens with `react-native-windows`.
